zeppelin-server/src/test/java/org/apache/zeppelin/rest/NotebookRestApiTest.java

```diff
@@ -164,6 +165,112 @@ public void testGetNoteByPathWithPathNotExist() throws IOException {
     post.close();
   }
 
+  @Test
+  public void testGetNoteRevisionHistory() throws IOException {
+    LOG.info("Running testGetNoteRevisionHistory");
+    String note1Id = null;
+    Notebook notebook = TestUtils.getInstance(Notebook.class);
+    try {
+      String notePath = "note1";
+      note1Id = TestUtils.getInstance(Notebook.class).createNote(notePath, anonymous);
```

Comment:
Use `notebook` here.

Reply:
Thanks for giving me the good advice!  I've changed all the code like this in this PR.